Mathlib/Order/LiminfLimsup.lean

```diff
@@ -617,6 +617,10 @@ theorem limsup_le_limsup {α : Type*} [ConditionallyCompleteLattice β] {f : Fil
   limsSup_le_limsSup hu hv fun _ => h.trans
 #align filter.limsup_le_limsup Filter.limsup_le_limsup
 
+theorem limsup_le_limsup' {α} [CompleteLattice β] {f : Filter α} {u v : α → β} (h : u ≤ᶠ[f] v) :
```

Comment:
My feeling is that autoparam works well with e.g. the `exact` tactic, but not the `apply` tactic. But, yes, I agree that the right solution is to fix autoparam instead of adding duplicate lemmas (and then delete the duplicate lemmas).

I have a dozen of examples of autoparam not firing. I will try to write some minimal examples for Zulip.

Reply:
For my issues with the automatic fill-in: I've opened a [Zulip thread](https://leanprover.zulipchat.com/#narrow/stream/239415-metaprogramming-.2F-tactics/topic/isBoundedDefault.20not.20triggering/near/450437091).

In the meantime, these specialized versions are removed. Better to work now with a wonky tool which will, hopefully, be fixed, than to have to delete lemmas (with all the downstream effects) later.